test/unit/data-generator.js

```diff
@@ -141,12 +123,12 @@ describe('Data generator', function() {
       var tests = generateTests(randomNumber, 'failed'),
           expected = [objectAssign({}, output.rootSuite, {tests: tests})];
 
-      expect(data).to.be.deep.equal(expected);
+      expect(data.length).to.be.deep.equal(expected.length);
```

Comment:
You can also use the [length](http://devdocs.io/chai/api/bdd/index#length-section) assertion.